sandbox/modules/identity/src/main/java/org/opensearch/identity/PrivilegesEvaluatorResponse.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.identity;
+
+import org.opensearch.action.admin.indices.create.CreateIndexRequestBuilder;
+
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+public class PrivilegesEvaluatorResponse {
```

Comment:
Can you add a brief javadoc for this and other newly introduced classes in this PR?

Reply:
Sure, some of these classes are boilerplate we can use when visiting authorization in future PRs. The `SecurityFilter` in this PR can be used for authorization, but in this case I added these when testing that tokens were being passed internally with transport requests by logging them out in the class. The test to verify that tokens are transported internally uses a `TransportMessageListener` to intercept messages when they are received at different nodes in the cluster.